### DISCUSSION TITLE
How to upgrade?

### DISCUSSION THREAD

**Discussion by Kasperdelasopa** [Koenkk/zigbee2mqtt#24473] [General]
I searched for a long time now put didn't find answer, so please help.

I want to update to a new version, as for now i had a couple of devices which could't be paired even if they are in the list of supported devices.

My Version of Z2M i 4 Years old so i wanted to upgrade.

I don't use Home-Assistent or docker. So, how cn i update my Z2M version running on my Raspberry?

**Comment by lux73** [comment]
> I searched for a long time now put didn't find answer, so please help.
> 

Linux general:

https://www.zigbee2mqtt.io/guide/installation/01_linux.html#for-later-update-zigbee2mqtt-to-the-latest-version

OpenHABian / Raspberry:

https://www.zigbee2mqtt.io/guide/installation/04_openhabian.html#update

make sure you a have a working Copy from your microSD Card if anything goes nuts ^^

**Reply by Kasperdelasopa** [reply]
Thank you very much!!!
